lib/ansible/modules/cloud/amazon/aws_codepipeline.py

```diff
@@ -96,8 +110,8 @@
               PollForSourceChanges: 'true'
               Branch: master
               # Generate token like this:
-              # https://docs.aws.amazon.com/codepipeline/latest/userguide/GitHub-rotate-personal-token-CLI.html
-              # GH Link: https://github.com/settings/tokens
+              # U(https://docs.aws.amazon.com/codepipeline/latest/userguide/GitHub-rotate-personal-token-CLI.html)
```

Comment:
Don't use `U(...)` here, since that will show up in the example and will not end up as a clickable link.